frontend/elements/src/entry/home/sections/whatsnew-section.ts

```diff
@@ -52,38 +55,37 @@ export class _ extends LitElement {
   height:201px;
 
  }
-
-
-
-
+ ::slotted([slot=contentpage]){
+  display:block;
+ margin-left:-110px;
+ }
 
     `];
   }
 
-  @property()
-  PATH_ARROW:string = ""; 
-
-
+   
 
   render() {
 
-    const {} = this;
-
-    const PATH_ARROW="icn-arrow.svg";
+    const { } = this;
 
+ 
 
     return html`
```

Comment:
unless I'm mistaken, this section is really more of a standard carousel thing and can be made generic / moved into core. It's fine to get it working here first and then move it out - but it should accept `Items` similar to AboutUs, generate the circle buttons, update arrow state, etc.